langchain/src/runnables/remote.ts

```diff
@@ -47,83 +54,110 @@ function isSuperset(set: Set<string>, subset: Set<string>) {
 function revive(obj: any): any {
```

Comment:
Could you double check that this client contains the same coverage as the list of well known types in python?

https://github.com/langchain-ai/langserve/blob/main/langserve/serialization.py#L54

Reply:
I think we are not handling agent types properly, will update.

Reply:
`ChatPromptValueConcrete` also does not exist in JS, will figure out what to do there